db/system_keyspace.cc

```diff
@@ -2603,6 +2634,73 @@ future<std::vector<system_keyspace::view_build_progress>> system_keyspace::load_
     });
 }
 
+future<service::vbc::vbc_tasks> system_keyspace::get_view_building_coordinator_tasks() {
+    using namespace service::vbc;
+    static const sstring query = format("SELECT * FROM system.{}", VIEW_BUILDING_COORDINATOR_TASKS);
```

Comment:
It's better to list the columns explicitly to avoid surprises when the schema changes.